basic-flatfield-correction/src/requirements.txt

```diff
@@ -1,4 +1,4 @@
-bfio==1.0.0
+bfio==1.0.1
```

Comment:
Why specify the version? Why not just always pull latest?

Reply:
Because there may be breaking changes or unexpected behavior. I know the plugin works with these specific set of packages using these defined versions. Also, bfio is not on PyPi, and if you do not have the same version of bfio that is included in the container, it will try to install a different version and throw an error.